pom.xml

```diff
@@ -1143,6 +1149,32 @@
         </executions>
       </plugin>
 
+      <!-- Use commit time for reproducible builds - https://maven.apache.org/guides/mini/guide-reproducible-builds.html -->
+      <plugin>
+        <groupId>pl.project13.maven</groupId>
+        <artifactId>git-commit-id-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>get-the-git-infos</id>
+            <goals>
+              <goal>revision</goal>
+            </goals>
+            <phase>initialize</phase>
+          </execution>
+        </executions>
+        <configuration>
+          <generateGitPropertiesFile>true</generateGitPropertiesFile>
+          <generateGitPropertiesFilename>${project.build.outputDirectory}/git.properties</generateGitPropertiesFilename>
+          <dateFormat>yyyy-MM-dd'T'HH:mm:ss'Z'</dateFormat>
+          <includeOnlyProperties>
+            <includeOnlyProperty>git.commit.time</includeOnlyProperty>
```

Comment:
we're only using this first property; should the others be included?

Reply:
this `git.properties` file gets added to jars. could this info be helpful?